spec/mailers/previews/user_mailer_preview.rb

```diff
@@ -13,9 +13,7 @@ def email_confirmation_instructions
   end
 
   def unconfirmed_email_instructions
-    UserMailer.unconfirmed_email_instructions(
```

Comment:
1. the `email_confirmation_instructions` preview still uses the old style of params (not `with`)
2. maybe separate follow-up, should we have a separate mailer preview for the new ReportMailer?

Reply:
1. Thanks, for catching that. Fixed that and a few others in 8ab8e35e3